specs/UI/Procedure.spec

```diff
@@ -0,0 +1,19 @@
+# Procedure
+
+## Admin should be able to upload and add procedure
+
+tags: snomed, ui
+
+* Login to Bahmni SNOMED location "General Ward" as a "receptionist"
+* Create new patient with random details and start an OPD visit
+* Create ValueSet for a procedure "consultation/procedure/valueSetHead"
+* Procedure created is uploaded in Bahmni
+* Logout and Login to Bahmni SNOMED location "General Ward" as a "doctor"
+* Open clinical tab
+* Open "Orders" Tab
+* Click on Procedure
+* Add Procedure
+* Open patient dashboard
+* Verify Procedure on patient clinical dashboard
```

Comment:
After verifying procedure, delete the automation procedure valueset on both snowstorm server and openMRS concept dictionary

Reply:
Done